src/test/unit/project-converter_test.ts

```diff
@@ -2951,6 +2963,52 @@ console.log('second script');
       });
     });
 
+    testName = 'copy over behavior comments properly';
+    test(testName, async () => {
+      setSources({
+        'test.html': `
+          <script>
+            /**
+             * This is a long and important comment.
+             * It has much info.
+             * @memberof Polymer This line should be removed
+             * @polymerBehavior
+             */
+            Polymer.MyBehavior = {};
+
+            /**
+             * This comment is also important, but its
+             * whitespace is handled better, because it doesn't
+             * have any lines that need filtering out.
+             * @polymer
+             * @mixinFunction
+             */
+            Polymer.MyMixinFunction = function() {};
+          </script>
+        `
+      });
+
+      assertSources(await convert(), {
+        'test.js': `
+/**
+             * This is a long and important comment.
```

Comment:
can we fix this formatting?

Reply:
The recast API is not well documented. I poked around in the debugger but didn't find anything that did exactly what we want. Hacked something together that seems to do all right though.